packages/strapi-plugin-users-permissions/config/functions/bootstrap.js

```diff
@@ -89,6 +89,15 @@ module.exports = async () => {
       secret: '',
       callback: '/auth/instagram/callback',
     },
+    vk: {
+      enabled: false,
+      icon: 'vk',
+      key: '',
+      secret: '',
+      callback: '/auth/vk/callback',
+      scope: ['email'],
+      custom_params: { v: '5.103', display: 'page' },
```

Comment:
Hello! This custom param is required?

Reply:
@lauriejim No, deleted it. I checked that it works without them.

Reply:
Okay great! Because of these params, the configuration modal is broken.
You will see some params that are not OK.
Can you please re-do the configuration screenshot, thank you.

Reply:
New configuration screenshot
[https://github.com/sulakin/strapi-examples/blob/master/login-react/assets/vk_settings__04.png](url)

Reply:
[New configuration screenshot](https://github.com/sulakin/strapi-examples/blob/master/login-react/assets/vk_settings__04.png)